lang/es.js

```diff
@@ -88,7 +88,7 @@ var language = {
       "reconnect-microbit": "Por favor reconecta el micro:bit e intentalo de nuevo.",
       "partial-flashing-disable": "Si el error persiste, intenta deshabilitar el flasheo rapido en las opciones beta.",
       "device-disconnected": "Dispositivo desconectado.",
-      "timeout-error":":Unable to connect to the micro:bit",
+      "timeout-error": "Unable to connect to the micro:bit",
```

Comment:
This entry is still **not** in the hr.js file

Reply:
sorry i missed that-- it should be now